backend/modules/pdf_processing/pipeline.py

```
@@ -263,16 +304,16 @@ async def process_full_with_progress(
                 "success": True,
                 "data": ai_result,
                 "session_id": session_id,
-                "scenario_id": scenario_id,
+                "simulation_id": simulation_id,
                 "message": "PDF parsed successfully"
             }
             
         except Exception as e:
             logger.error(f"[PIPELINE] Full processing failed: {str(e)}")
```

Ruff (0.14.8)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)